pkg/util/mount/nsenter_mount.go

```diff
@@ -323,3 +323,19 @@ func (mounter *NsenterMounter) PrepareSafeSubpath(subPath Subpath) (newHostPath
 func (mounter *NsenterMounter) SafeMakeDir(pathname string, base string, perm os.FileMode) error {
 	return doSafeMakeDir(pathname, base, perm)
 }
+
+func (mounter *NsenterMounter) GetMountRefs(pathname string) ([]string, error) {
+	realpath, _, err := mounter.ne.KubeletPath(pathname)
+	if err != nil {
+		return nil, err
+	}
+	return getMountRefsByDev(mounter, realpath)
```

Comment:
I thought nsenter getMountRefs actually returns mounts based on host paths.  So we shouldn't need to use KubeletPath here?

Reply:
It needs `KubeletPath`, because it needs the path name on the host after the evaluation of symbolic links to search mount references.

`KubeletPath` now returns two paths for `/mnt/to/link` (`/mnt/to/link` links to `/mnt/to/dir`):

- the path name on the host after the evaluation of symbolic links (e.g. /mnt/to/dir on the host)
- the path name that can be accessed by containerized kubelet, after evaluating symlinks on the host. (e.g. /rootfs/mnt/to/dir in container)

Reply:
Ah I see.  I got confused by the function name.  What if it's split up into two functions?  One for evaluating the symlink, and one for getting the kubelet path?

Reply:
Good idea! Done into https://github.com/kubernetes/kubernetes/pull/62903/commits/6d484fca6606af1a712ce092d5a7d5433e48c712.